tests/test_leftovers.py

```diff
@@ -0,0 +1,294 @@
+from indexd.blueprint import get_record
+from tests.test_client import get_doc
+import pytest
+from indexd.index.errors import NoRecordFound as IndexNoRecordFound
+from indexd.dos.blueprint import get_dos_record
+import json
+
+
+def test_get_record_call_dist_get_record(client, user):
+    # test get_record correctly
+    data = get_doc()
+    res = client.post("/index/", json=data, headers=user)
+    assert res.status_code == 200
+    rec = res.json
+    response = get_record(rec["did"])
+    record = response[0].json
+    assert record["urls"][0] == "s3://endpointurl/bucket/key"
+    assert record["size"] == 123
+
+    # test call dist_get_record for unposted record
+    fake_did = "testprefix:d96bab16-c4e1-44ac-923a-04328b6fe78f"
+    with pytest.raises(IndexNoRecordFound):
+        get_record(fake_did)
```

Comment:
also use the `GET /<GUID>` endpoint here, and i think you'll have to check the status code instead of looking for `IndexNoRecordFound`